onnxruntime/core/providers/cuda/reduction/reduction_ops.cc

```diff
@@ -241,6 +302,77 @@ Status ReduceKernel<allow_multi_axes>::ComputeImpl(OpKernelContext* ctx, cudnnRe
   return Status::OK();
 }
 
+template <>
+template <>
```

Comment:
you might want to add comments on why you need to handle int32 explicitly

Reply:
also why are there two 'template<>' lines?

---
In reply to: [338331298](https://github.com/microsoft/onnxruntime/pull/2108#discussion_r338331298) [](ancestors = 338331298)